### ISSUE TITLE
Create HomePage CSS

### ISSUE THREAD

**Issue by BenJensen2** [Team-Programming/Platform#4]
Add Styling to the homepage.  Try things out, make it look good and see what happens!

**Comment by Shalini469717** [comment]
@BenJensen2 Can I work on this? I am currently learning CSS

**Comment by BenJensen2** [comment]
@Shalini469717 Yes, feel free to design it however you'd like. Please reference the Vision document. If anything is unclear, you can message me but feel free to lean on the "You get to decide and design it" side over the "It has to be a specific way."

**Comment by Shalini469717** [comment]
Sure!
